### ISSUE TITLE
Include discussion emails in activity log

### ISSUE THREAD

**Issue by kshuttle** [pkp/pkp-lib#7074]
**Describe the problem you would like to solve**
Currently, emails created as part of a discussion are not included in the journal's activity log. The activity log records messages and corresponding files that are sent when someone new (such as a copyeditor or layout editor) is assigned to a submission. However, if a new discussion is begun during that stage of the process for questions or files to go back and forth, these emails are only logged in the discussion area while a “file uploaded” is logged in the activity log (but not the corresponding email for context).

The following screenshots were provided by a user who requested this feature in the Community Forum:

Here, a new discussion was generated at the end of March:
![image](https://user-images.githubusercontent.com/59741486/119208044-5e2e2100-ba55-11eb-9f13-a73a98131030.png)

But this is not reflected in the activity log, where the last activity is shown as being in January.

![image](https://user-images.githubusercontent.com/59741486/119208161-b402c900-ba55-11eb-87d9-46be31412832.png)

Similarly, these screenshots show an email was generated when a copyeditor was assigned, which shows up in both discussion and activity log; however the follow-up discussion emails are not registered in the activity log:
![image](https://user-images.githubusercontent.com/59741486/119208189-d399f180-ba55-11eb-8d4a-83484cf96f38.png)

![image](https://user-images.githubusercontent.com/59741486/119208195-dc8ac300-ba55-11eb-93e6-81ceebf15d1f.png)

**Describe the solution you'd like**
Emails that are generated in the discussion areas should be represented in the activity log as that is a logical place to look to track the full progression of a submission.

**Who is asking for this feature?**
This was [requested by a user in the Community Forum](https://forum.pkp.sfu.ca/t/reviewer-copyeditor-production-discussions-not-represented-in-activity-log/67781).

**Comment by hxo82** [comment]
@kshuttle @NateWr 
Not sure if this is a component of this particular request, but I'd love to also suggest that when you start a discussion, the email that ends up in the recipient's inbox contains the actual email sent through the discussion rather than a notification to log in and see a message. I think this would streamline communications, and you know for sure the recipient receives the message you intended for them. Our editors feel like a notification of a message is an extra unnecessary step for busy researchers. Others may feel differently. Thanks everyone!

**Comment by NateWr** [comment]
Hi @hxo82, you can find an issue filed for that here: https://github.com/pkp/pkp-lib/issues/6222.

**Comment by hxo82** [comment]
Thanks @NateWr !

**Comment by amandastevens** [comment]
+1 from a PKP PS client

**Comment by NateWr** [comment]
@Vitaliy-1 this is implemented in `main`. However, there's a problem with the mailable footers. When the email body is logged, it includes the footer:

![mailable-footer-log](https://user-images.githubusercontent.com/2306629/201909843-297ecbce-7354-4e8f-bdde-df2c63442e90.png)

This is a problem because it exposes the unsubscribe link to everyone. This is happening in `SubmissionEmailLogDAO`:

https://github.com/pkp/pkp-lib/blob/d5eef799c057082619b3c37d269d855084f9f3e7/classes/log/SubmissionEmailLogDAO.php#L79

If the `Mailable::$footer` property were public, we could simply overwrite it before logging:

```php
$mailableCopy = clone $mailable;
$mailableCopy->footer = '';
$entry->setBody($mailable->render());
```

But maybe you want to keep it protected, in which case we would need an alternative method to `render()` that just gets the body without the footer. Does something already exist?

**Steps to Reproduce**
1. Create a new discussion with multiple participants.
2. Click Activity Log
3. Click the expand arrow for the "email has been sent" entry.
4. Click View Email.

(Don't worry about the log saying `An email has been sent: {$subject} `. That will be fixed in #5716.

**Comment by Vitaliy-1** [comment]
PRs
pop-lib: https://github.com/pkp/pkp-lib/pull/8614
tests: https://github.com/pkp/ojs/pull/3748
